docs/guides/parallel-scraping/parallel-scraping.mdx

```diff
@@ -21,54 +18,45 @@ At the time of writing this guide (December 2023), request locking is still an e
 
 :::
 
-In this guide, we will walk through how you can turn your single scraper into a scraper that can be parallelized and run in multiple instances. This guide
-assumes you've read and walked through our [introduction guide](../introduction/setting-up) (or have a fully-fledged scraper already built), but if you haven't done so yet,
-take a break, go read through all that, and come back. We'll be waiting...
+In this guide, we will walk you through how you can turn your single scraper into a scraper that can be parallelized and run in multiple instances. This guide assumes you've read and walked through our [introduction guide](../introduction/setting-up) (or have a fully-fledged scraper already built), but if you haven't done so yet, take a break, go read through all that, and come back. We'll be waiting...
 
 *Oh, you're back already! Let's proceed in making that scraper parallel!*
 
 ## Things to consider before parallelizing
 
 Before you rush ahead and change your scraper to support parallelization, take a minute to consider the following factors:
 
-- do you plan on scraping so many pages that you need to parallelize your scraper?
-  > for example, if your scraper goes across few pages, you probably don't need parallelization  
-  > but if you scrape a lot of pages, or you scrape pages that take a long time to load, you might want to consider parallelization
-- can you parallelize your scraper while not overloading the source website?
-  > for example, if you scrape a website that has a lot of traffic, you don't want to add to that traffic by running multiple scrapers in parallel
-  > as that might cause the website to go down for all its users
-- do you have the resources available to run multiple scrapers in parallel?
-  > when running locally, depending on your scraper type, do you have enough CPU and RAM available to sustain multiple scrapers running in parallel  
-  > when running in the cloud, will the extra speed from parallelization be worth the extra cost of running multiple scrapers in parallel?
+- Do you plan on scraping so many pages that you need to parallelize your scraper?
+  > For example, if your scraper goes across a few pages, you probably don't need parallelization<br/>
```

Comment:
i still dont understand why you want the line break here, imo there should be none, if you want to have two paragraphs, you need an extra blank line instead of the `<br>` tag. maybe it should be a nested bullet list instead of quotes?

Reply:
Mostly for the way it looks (see https://github.com/apify/crawlee/pull/2219#discussion_r1438256142 - response to @TC-MO's review above yours).

Jus to get the ball rolling, I'll make it a nested list, it's a niche thing I'd rather not let hold back this PR

Reply:
If it looks better, feel free to keep it, it just feels weird